src/NHibernate.Test/CacheTest/QueryCacheFixture.cs

```diff
@@ -90,5 +91,48 @@ public void QueryCacheWithScalarReturn()
 				Assert.That(result, Is.EqualTo(200012), "Unexpected cached result");
 			}
 		}
+
+		[Test]
+		public void QueryCacheWithAliasToBeanTransformer()
+		{
+			const string query = "select s.id_ as Id from Simple as s;";
```

Comment:
This causes the test to fail for Oracle, which does not support statements ending with a semi-colon. Can you rewrite the test with a HQL query instead? It would let NHibernate handles this.